docs/general/session.md

```diff
@@ -35,4 +35,14 @@ backends can link the two sessions.
 <!-- END AUTOGENERATED TEXT -->
 <!-- endsemconv -->
 
+## Session Change Event
+
+![Experimental](https://img.shields.io/badge/-experimental-blue)
+
+When a session changes (due to time-based expiry or another mechanism), an event MAY be emitted.
+
+The emitted event that represents a session change MUST have the `event.name=session.change`.
+The event body MUST be empty and the attributes MUST include both the `session.id` and `session.previous_id`
```

Comment:
Should we allow `session.id` to be null to denote the change resulting in a session expiring but no new session beginning?

Reply:
Maybe. What's the use case? Session end? Seems like we could have a separate event then, keeping parity with a hypothetical session start event.

Reply:
The use case is when the "change" doesn't lead to a new session - just the ending of the previous one, like when there is a requirement for a new session that is not fulfilled.

For instance, if a session times out, but the app is in the background, and no telemetry will be recorded while that is the case, do we want to generate a new `session.id`?

An alternative is to use some non-null stub to indicate that there's no session, e.g. empty-string. If we go down that route, we should define what that non-null stub should be.

Reply:
I agree with having separated events for single cases like session change/end/start.